src/app/screens/LNURLPay.tsx

```diff
@@ -106,13 +106,20 @@ function LNURLPay(props: Props) {
         comment, // https://github.com/fiatjaf/lnurl-rfc/blob/luds/12.md
         payerdata, // https://github.com/fiatjaf/lnurl-rfc/blob/luds/18.md
       };
-      const { data: paymentInfo } = await axios.get<LNURLPaymentInfo>(
-        details.callback,
-        {
+
+      let paymentInfo,
+        paymentRequest = null;
+
+      try {
+        const request = await axios.get<LNURLPaymentInfo>(details.callback, {
           params,
-        }
-      );
-      const { pr: paymentRequest } = paymentInfo;
+        });
+        paymentInfo = request.data;
+        paymentRequest = paymentInfo.pr;
+      } catch (e) {
+        alert("Payment aborted: Could not fetch invoice.");
```

Comment:
The LNURL server can also return an "error" in the response. 
Right now we do not handle this one either. 

```
{"status":"ERROR", "reason":"error details..."}
```

I guess checking for the status "error" and then doing an alert(reason) would be good?

Reply:
Thanks for the hint, I just added the reason to the alert displayed to the user (if available). 

![image](https://user-images.githubusercontent.com/100827540/161285039-608e2cf8-bf08-4634-95e9-796ff3767501.png)

Reply:
can you check the spec. I think that the service can also respond with a HTTP 200 BUT have an error field.